setup.py

```diff
@@ -79,7 +80,7 @@ def required(requirements_file):
             'mycroft-audio=mycroft.audio.__main__:main',
             'mycroft-echo-observer=mycroft.messagebus.client.ws:echo',
             'mycroft-audio-test=mycroft.util.audio_test:main',
-            'mycroft-enclosure-client=mycroft.client.enclosure.__main__:main',
+            'mycroft-enclosure-client=ovos_PHAL.__main__:main',
```

Comment:
Does this break Old Stuff? Something seems weird about it, but it's the obvious change to make.

Reply:
This would overwrite any previous definition, but presumably the entry points serve the same function, so I'd say this is valid..